File: User/src/Entity/Commande.php
<?php

namespace App\Entity;

use App\Repository\CommandeRepository;
use Doctrine\ORM\Mapping as ORM;
use Doctrine\ORM\Mapping\JoinColumn;
use Symfony\Component\Serializer\Annotation\Groups;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * @ORM\Entity(repositoryClass=CommandeRepository::class)
 * @method getCommandemontant()
 */
class Commande
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     * @Groups("post:read")
     */
    private $id;

    /**
     * @ORM\Column(type="decimal", precision=10, scale=0)
     *  @Groups("post:read")
     * @Assert\NotBlank(message="montant obligatoire")

     */
    private $montant;

    /**
     * @ORM\Column(type="date")
     *  @Groups("post:read")
     */
    private $date;

    /**
     * @ORM\Column(type="integer")
     * @Groups("post:read")
     * @Assert\NotBlank(message="quantiter obligatoire")
     */
    private $qte;

    /**
     * @ORM\Column(type="string", length=20, options={"default" : "encour"})
     */
    private $etat;

    /**
     * @ORM\ManyToOne(targetEntity=Produit::class, inversedBy="commandes")
     * @JoinColumn(name="idproduit", referencedColumnName="idproduit")
     */
    private $produit;


    public function getId(): ?int
    {
        return $this->id;
    }

    public function getMontant(): ?string
    {
        return $this->montant;
    }

    public function setMontant(string $montant): self
    {
        $this->montant = $montant;

        return $this;
    }

    public function getDate(): ?\DateTimeInterface
    {
        return $this->date;
    }

    public function setDate(\DateTimeInterface $date): self
    {
        $this->date = $date;

        return $this;
    }
    public function getQte(): ?int
    {
        return $this->qte;
    }
    public function setQte(int $qte): self
    {
        $this->qte = $qte;

        return $this;
    }

    public function getEtat(): ?string
    {
        return $this->etat;
    }

    public function setEtat(string $etat): self
    {
        $this->etat = $etat;

        return $this;
    }
    public function __toString()
    {
        return $this->montant;
    }

    public function getProduit(): ?Produit
    {
        return $this->produit;
    }

    public function setProduit(?Produit $produit): self
    {
        $this->produit = $produit;

        return $this;
    }



}

File: codename-project/src/com/mycomany/entities/Commande.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycomany.entities;

import java.util.Date;

/**
 *
 * @author lenovo
 */
public class Commande {
    public int id,qte;
    public float montant;
    public String date;
    public String etat;

    public Commande() {
    }

    public Commande(int qte, float montant, String etat) {
        this.qte = qte;
        this.montant = montant;
        this.etat = etat;
    }

    
    
    
    public Commande(int id, int qte, float montant, String date, String etat) {
        this.id = id;
        this.qte = qte;
        this.montant = montant;
        this.date = date;
        this.etat = etat;
    }

    public Commande(int qte, float montant, String date, String etat) {
        this.qte = qte;
        this.montant = montant;
        this.date = date;
        this.etat = etat;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getQte() {
        return qte;
    }

    public void setQte(int qte) {
        this.qte = qte;
    }

    public float getMontant() {
        return montant;
    }

    public void setMontant(float montant) {
        this.montant = montant;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getEtat() {
        return etat;
    }

    public void setEtat(String etat) {
        this.etat = etat;
    }

   

  


    
    
    
    
    
}
